package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@fieldguide/pipeline",
   "description": "A toolkit to easily build synchronous process pipelines in TypeScript/JavaScript",
-  "version": "0.1.0",
+  "version": "1.2.0",
```

Comment:
We don't need to set this. It automatically gets updated when we publish to NPM…

<img width="728" alt="Screenshot 2024-10-21 at 11 35 27 AM" src="https://github.com/user-attachments/assets/70995ace-a239-4942-afb3-a060cd82be66">

We could even set it to something like `0.0.1` to make it really obvious that we don't need to update it.